Maestro/subscriptions.json

```diff
@@ -754,6 +754,7 @@
       "delay": "00:10:00",
       "actionArguments": {
         "vsoBuildParameters": {
+          "vsoSourceBranch": "master",
```

Comment:
This should actually be an `actionArguments` property:

https://github.com/dotnet/versions/blob/b32284828bce0b1e357abdc8c29b78f8924d6013/Maestro/subscriptions.json#L131-L134